message-queue-lib/project.clj

```diff
@@ -57,10 +57,10 @@
              ;; level directory.
              :lint {:source-paths ^:replace ["src"]
                     :test-paths ^:replace []
-                    :plugins [[jonase/eastwood "0.2.5"]
-                              [lein-ancient "0.6.15"]
-                              [lein-bikeshed "0.5.0"]
-                              [lein-kibit "0.1.6"]]}
+                    :plugins [[jonase/eastwood "1.4.2"]
+                              [lein-ancient "0.7.0"]
```

Comment:
Do we just want to update these plugins, or do we want to go ahead and replace them with tools that are under active development?

Reply:
being last day of the sprint/pi, I'm inclined to say no, but I can incorporate this into the future tickets? was it pretty simple?

Reply:
I'll see what I can do this PI to update some of these things. I think we are all pretty close to voting to just drop eastwood.